apps/daimo-mobile/src/model/account.ts

```diff
@@ -87,6 +88,10 @@ export type Account = {
 
   /** Linked accounts (via mutual sig) for rich profiles, eg Farcaster. */
   linkedAccounts: LinkedAccount[];
+  /** Invite link and it's status */
```

Comment:
micro nit: its

more importantly, is this the invite link the user came in on, or an invite link they can give out?

Reply:
added comment